tools/ibc-test/src/tests/mod.rs

```diff
@@ -27,24 +24,37 @@ fn matrix_test_within_axon_and_ckb() -> Result<(), Error> {
             {"PACKET_TYPE_ARGS", hex::encode(PACKET_TYPE_ARGS)},
             {"CLIENT_TYPE_ARGS", hex::encode(CLIENT_TYPE_ARGS)},
         );
-        if value == "ckb,ckb" {
-            packet_run = true;
-        }
     }
-    run_arbitrary_binary_channel_test(&ChannelTest::new(&CKB4IbcPacketTest::new(packet_run)))
+    Ok(())
+}
+
+#[test]
+fn test_channel() -> Result<(), Error> {
+    init_envs()?;
+    run_arbitrary_binary_channel_test(&ibc::channel::ChannelTest::new())
+}
+
+#[test]
+fn test_ckb_packet() -> Result<(), Error> {
+    init_envs()?;
+    let value = std::env::var("ACCOUNT_PREFIXES")
+        .map_err(|e| eyre::eyre!("no ENV entry \"ACCOUNT_PREFIXES\": {e}"))?;
+    if value == "ckb,ckb" {
+        log::info!("Run ckb packet tests for {}", value);
+        run_arbitrary_binary_channel_test(&ckb::packet::CKB4IbcPacketTest::new())
```

Comment:
I think the packet test here is dependent with channel test above, because the test goal of success of packet is based on the successful created channel and connection cells on CKB

Reply:
The channel / connection creation is done in `run_arbitrary_binary_channel_test`, `ChannelTest` just check the cells is created. so the packet test isn't dependent on `ChannelTest`.